src/Constants/MENULIST.js

```diff
@@ -0,0 +1,65 @@
+//메뉴판 데이터
+const MENULIST = [
```

Comment:
```suggestion
const MENU_LIST = [
```
`MENU_LIST`로 스네이크 케이스를 적용해봐도 좋을 것 같아요! 

하드코딩 하느라 고생하셨겠군요... 코파일럿이라도 있으면 그나마 괜찮은데 최종 코테에서는 코파일럿이 금지라 아쉽네요😭